### ISSUE TITLE
error when displaying .jpg from web

### ISSUE THREAD

**Issue by mtshrmn** [eXeC64/imv#306]
Every time I'm trying to use imv with curl as documented in the readme I get the following error:
```sh
suerflowz in ~
➜ curl https://i.imgur.com/TWI1bid.jpg | imv -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 59285  100 59285    0     0   116k      0 --:--:-- --:--:-- --:--:--  116k
double free or corruption (!prev)  # (here is where I press q) 
fish: Process 184411, “imv” “curl https://i.imgur.com/TWI1bi…” terminated by signal SIGABRT (Abort)
```
This works with everything else, just not with jpg from curl. 

---
System info:
- imv version: v4.2.0
- os: 5.9.14-arch1-1
- Xorg
- shell: fish v3.1.2 (occurs with bash v5.1.4 as well)

P.S. This project is amazing, thank you everyone for creating and developing it!

**Comment by mtshrmn** [comment]
I've fixed my specific problem by using `imv (curl https://i.imgur.com/TWI1bid.jpg | psub)`. But the issue still remains.

**Comment by eXeC64** [comment]
Fixed by #334
